vcpkg.json

```diff
@@ -0,0 +1,62 @@
+{
+    "name": "freerdp",
+    "version-string": "3.3.0",
+    "description": "A free implementation of the Remote Desktop Protocol (RDP)",
+    "homepage": "https://github.com/FreeRDP/FreeRDP",
+    "license": "Apache-2.0",
+
+    "dependencies": [
+        "zlib",
+		"openssl"
+    ],
+
+    "default-features": [
+    ],
+
+    "features": {
+		"all": {
+			"description": "Enable all features",
+            "dependencies": [
+				"openssl",
+				"openh264",
+				"opus",
+				"ffmpeg",
+				"cjson",
+				"libusb",
+				"opencl",
+				"cairo",
+				"uriparser",
+				"sdl2",
+				"sdl2-ttf"
```

Comment:
missing `sld2-image`

Reply:
webp,png,jpeg is a dependency of sdl2-image. I only include sdl2-image. vcpkg progress it dependency.